### PR TITLE
option to pass --file argument for docker build

### PR DESCRIPTION
**Pull request by sajithmohan**
Sometime we have dockerfile in different path than root 
for example [example](https://github.com/onrunning/order-hub-importer/blob/06d568857dbe9e884ccf1967ac7c3cb71757e31b/docker/live/Dockerfile).

This PR will enable to pass optional argument to use as value for [--file](https://github.com/onrunning/order-hub-importer/blob/06d568857dbe9e884ccf1967ac7c3cb71757e31b/docker/live/Dockerfile) arg in docker build

### CODE REVIEW

**Review by amalnaeem4 — APPROVED** [review]

File: .github/workflows/build_push_image.yaml [hunk 1]
```diff
@@ -6,14 +6,19 @@ on:
...
-        type: string 
+        type: string
       docker_path:
         required: false
         default: .
-        type: string   
+        type: string
+      docker_file_path:
+        required: false
+        default: ./Dockerfile
```
Comment (ssoyoung) [inline .github/workflows/build_push_image.yaml:19]:
```suggestion
        default: .
```

Comment (ssoyoung) [reply .github/workflows/build_push_image.yaml:19]: It should be specified only path (without filename), otherwise it'll fail 😢

Comment (sajithmohan) [reply .github/workflows/build_push_image.yaml:19]: No, this is default for --file, I think you are mentioning the context path which is accepted as `docker_path` @ssoyoung

Comment (sajithmohan) [reply .github/workflows/build_push_image.yaml:19]: `docker build -f Dockerfile.debug .` 
The final argument  (`.`) accepted with `docker_path`, new argument `-f` accepted in `docker_file_path`

**Review by ekpangmichael — APPROVED** [review]

**Review by ssoyoung — APPROVED** [review]

File: .github/workflows/build_push_image.yaml (hunk 1 above)
Comment (ssoyoung) [reply .github/workflows/build_push_image.yaml:19]: ah true true 👍🏼

**Pull request merged**